src/components/PageTitle/__snapshots__/PageTitle.test.js.snap

```diff
@@ -0,0 +1,12 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<PageTitle /> should match the snapshot 1`] = `
+<div>
```

Comment:
Not a big deal, but in the snapshot you see this `div`, which is not present in `PageTitle` component
This div is used internally by the library, and to make the snapshot shorter and cleaner we can do it not from the whole `container` object, but from it's `firstChild`

```
const { container: { firstChild } } = render(..)
expect(firstChild).toMatchSnapshot()
```